src/bluesearch/entrypoint/database/convert_pdf.py

```diff
@@ -63,22 +67,38 @@ def init_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         help="The port of the GROBID server.",
     )
     parser.add_argument(
-        "input_pdf_path",
+        "input_path",
         type=pathlib.Path,
-        metavar="INPUT-PDF-PATH",
-        help="The path of the input PDF file.",
+        metavar="INPUT-PATH",
+        help="The path to a single PDF file or a directory with many PDF files.",
     )
     parser.add_argument(
-        "output_xml_path",
+        "-o",
+        "--output-dir",
         type=pathlib.Path,
-        metavar="OUTPUT-XML-PATH",
-        help="The path of the output XML file.",
+        metavar="OUTPUT-DIR",
+        help="""
+        The output directory where the XML file(s) will be saved. If not
+        provided the output files will be placed in the same directory as
+        the input files.
+        """,
+    )
+    parser.add_argument(
+        "-w",
+        "--num-workers",
+        type=int,
+        default=5,
```

Comment:
Is there a rational behind the number 5?

Reply:
Good catch, absolutely not actually. We just put a random number in during dev and never changed it. I'm not sure what's a good default though. Maybe 1?